src/GitHubExtension/Widgets/GitHubRepositoryWidget.cs

```diff
@@ -46,6 +46,19 @@ public string GetUnescapedIssueQuery()
         return Uri.UnescapeDataString(GetIssueQuery()).Replace('+', ' ');
     }
 
+    // If the user changed the URL after clicking submit and clicked
+    // saved just after, we change it back to what was before to not corrupt our saved data.
+    private void CorrectUrl()

```

Comment:
The comments sound like a race-condition.  What is happening between "Submit" and "Save"?

Reply:
They are separate buttons in the widget, so unlikely that the user could click them both fast enough to cause an issue.